src/Time/TimeSteppers/RungeKutta4.cpp

```diff
@@ -0,0 +1,66 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Time/TimeSteppers/RungeKutta4.hpp"
+
+#include <cmath>
+#include <limits>
+
+#include "ErrorHandling/Assert.hpp"
+#include "Time/TimeStepId.hpp"
+
+namespace TimeSteppers {
+
+uint64_t RungeKutta4::number_of_substeps() const noexcept { return 5; }
```

Comment:
change to `4`

Reply:
My current workaround actually made me change this back to 5, since I added an extra substep to get dense output working.